docker-compose.yml

```diff
@@ -14,10 +14,3 @@ services:
     entrypoint: /bin/bash
     stdin_open: true
     tty: true
-    environment:
-      # NB: default values (they're blank here) — if you don't use the ${VAR:-default} syntax, Docker will be blaming about env vars not set..
-      - API_KEY="${API_KEY:-}"
-      - SECRET="${SECRET:-}"
-      - WALLET_ADDRESS="${WALLET_ADDRESS:-}"
-      - PRIVATE_KEY="${PRIVATE_KEY:-}"
```

Comment:
Those variables were originally introduced by a guy from TheOcean, I believe. They are used in some TheOcean-related examples. I decided to leave them temporarily, but I agree that there is no real need to put them there.